src/edu/csus/ecs/pc2/core/report/JudgementReport.java

```diff
@@ -76,6 +81,23 @@ public void printHeader(PrintWriter printWriter) {
         printWriter.println();
         printWriter.println(getReportTitle() + " Report");
     }
+    
+    public void printRejectINI(PrintWriter printWriter, IInternalContest inContest) {
```

Comment:
See previous review comment.  Adding this method doesn't seem to me to fall under Issue #119.

Reply:
I did not make this change.